README.md

```diff
@@ -1,61 +1,84 @@
 # dungeon-revealer [![Build Status](https://travis-ci.org/apclary/dungeon-revealer.svg?branch=master)](https://travis-ci.org/apclary/dungeon-revealer)
 
-![alt text](https://user-images.githubusercontent.com/14338007/57970988-634cc700-7988-11e9-8cae-dfcc685cc848.png "Preview Image")
 
 A web app for tabletop gaming to allow the game master to reveal areas of the game map to players.
 
 ## What the DM Sees
 
-![alt text](http://apclary.github.io/dungeon-revealer/img/example_dm_1.jpeg "DM's view")
+![alt text](https://user-images.githubusercontent.com/9096667/60119974-6b491500-9745-11e9-9e13-6c52b20b605d.png "DM's View")
 
 You can protect the DM area by setting a password.
 
 ## What the players see
 
-![alt text](https://apclary.github.io/dungeon-revealer/img/example_player_1.jpeg "Player's view")
+![alt text](https://user-images.githubusercontent.com/9096667/60120496-86685480-9746-11e9-903d-87b23369451c.png "Player's view")
 
 ## Getting Started
 
 ### Getting the app
 
 The easiest way to use dungeon-revealer is to download the app from the [releases](https://github.com/apclary/dungeon-revealer/releases) page here on github. There is also a [docker image](https://hub.docker.com/r/apclary/dungeon-revealer) that is kept up to date with the releases in this repository.
 
-#### Prebuilt app 
+#### Prebuilt app
 
 **Download the app for your system from the [releases page](https://github.com/apclary/dungeon-revealer/releases).**
 We provide builds for Windows, Linux, and OSX.
 
-Either double click the app or open a command prompt in the same directory as the app and run from there. 
 Running from the command prompt will present connection information and some debugging.
```

Comment:
Idk about other platforms but on OSX it will always show the app terminal, whether you open it with a double click or via the terminal

Reply:
It doesn't open the terminal for Linux. It does open it for Windows. 

It may be reasonable to assume that Linux users know enough to fend for themselves. Lots of other projects do that.

Reply:
See c2f996d